install.sh

```diff
@@ -57,33 +83,42 @@ chroot_install() {
         || exit 1
     chmod +x ./alpine-chroot-install || exit 1
     ./alpine-chroot-install -d /alpine-v310 -b v3.10
-    /alpine-v310/enter-chroot install.sh
+    /alpine-v310/enter-chroot ./install.sh
+}
+
+check_os() {
+    if [ -f "/etc/alpine-release" ]; then
+        alpine_install || exit 1
+    elif [ -x "$(command -v apt-get)" ]; then
+        debian_install || exit 1
+    else
+        echo "Not an Alpine or Debian-based Linux system"
+        echo
+        echo "Attempting to create Alpine chroot"
+        chroot_install || exit 1
+    fi
 }
 
-check_vm() {
+check_root() {
+if ! [ $(id -u) = 0 ]; then
```

Comment:
My IDE shouts on me whenever I do that :P 

<img width="361" alt="MacOS 2019-09-11 at 02 21 35" src="https://user-images.githubusercontent.com/526657/64643808-6c800900-d43b-11e9-8b95-60e21111fc6a.png">

Plus `!=` might be easier to understand, and is less likely to fail in Bourne Shell.

```shell
if [ "$(id -u)" != "0" ]; then
   echo "Error: You must be root to install"
   exit 1
fi
```

Reply:
Plius please indent content of the fn.